dev/com.ibm.ws.cdi.ejb_fat/test-applications/newEjbTimer.war/resources/WEB-INF/beans.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://java.sun.com/xml/ns/javaee"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="
+      http://java.sun.com/xml/ns/javaee
+      http://java.sun.com/xml/ns/javaee/beans_1_0.xsd">
+
+</beans>
```

Comment:
What's the intention of this beans.xml?

I think it's using the CDI-1.0 schema and ends effectively setting `bean-discovery-mode=all`?

Reply:
Beans.xml was unnecessary and has been removed.